core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -271,7 +271,7 @@ public function from($identifier): self
     /**
      * Sets a size configuration that will be applied to the resource.
      *
-     * @param int|string|array|PictureConfiguration $size A picture size configuration or reference
+     * @param int|string|array|PictureConfiguration|null $size A picture size configuration or reference
```

Comment:
Don't you have to adjust the property phpDoc as well? Currently it is `@var mixed|null`, which is definitely wrong, because `mixed` already includes `null`. So either you remove the `@param` annotation completely (then `mixed` is implied) or you adjust the `@var` annotation accordingly.

Reply:
Ah I missed that. Fixed in 74310e3eabc7a8243fe6c47246d31a37e50e46f9.